recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "bokeh" %}
-{% set version = "3.1.1" %}
+{% set version = "3.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bokeh-{{ version }}.tar.gz
-  sha256: ba0fc6bae4352d307541293256dee930a42d0acf92e760c72dc0e7397c3a28e9
+  sha256: cdb3a8c3109d8d91a2814f4cddea045c14d049021f7fb6ce2a6b2c15e224bb17
 
 build:
   number: 0
-  skip: true  # [py<38]
+  skip: true  # [py<39]
   script:
     - {{ PYTHON }} -m pip install . -vv
```

Comment:
All good! My only suggestion is to perhaps amend the `script` to 
`script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv`

Reply:
@Arishamays1 I've reformulated your proposed change as a GitHub `suggestion` and it was applied by @philippjfr